packages/browser/__tests__/login/oidc/redirectHandler/ImplicitRedirectHandler.spec.ts

```diff
@@ -89,15 +91,15 @@ describe("GeneralRedirectHandler", () => {
   });
 
   describe("handle", () => {
-    it("returns an unauthenticated session on non-redirect URL", async () => {
-      const authCodeRedirectHandler = getGeneralRedirectHandler();
-      const mySession = await authCodeRedirectHandler.handle("https://my.app");
-      expect(mySession.isLoggedIn).toEqual(false);
-      expect(mySession.webId).toBeUndefined();
+    it("throws on non-redirect URL", async () => {
+      const redirectHandler = getImplicitRedirectHandler();
+      await expect(redirectHandler.handle("https://my.app")).rejects.toThrow(
+        "ImplicitRedirectHandler cannot handle [https://my.app]"
```

Comment:
Can this message explain 'why' it can't handle that valid IRI? (There's nothing worse than a message like "Internal Server Error" :( !)